word_scripts/wordfind_site.py

```diff
@@ -0,0 +1,26 @@
+import requests
+from bs4 import BeautifulSoup
+
+def wordFindSite ():
```

Comment:
Also there is no space after funciton name and the brackets ex. do foo(), not foo ()